packages/material-ui/package.json

```diff
@@ -58,7 +58,7 @@
     "jss-vendor-prefixer": "^7.0.0",
     "normalize-scroll-left": "^0.1.2",
     "popper.js": "^1.14.1",
-    "prop-types": "^15.6.0",
+    "prop-types": "^15.7.2",
```

Comment:
`PropTypes.elementType` was introduced in `15.7.0`. Can we lose a bit the version requirement?
https://github.com/facebook/prop-types/blob/master/CHANGELOG.md

Reply:
We already had 15.7.1 in the lockfile anyway and 15.7.2 had an important bug fix for browserify build. I would rather propagate that to downstream packages. The only time this would cause duplication (assuming this was bad) is when someone actually pins their `prop-types` to `"prop-types": "15.7.1"` and I don't see any reason for that.

Reply:
Ok, I was looking for potential dependency deduplication for our users.

Reply:
That can always happen which is why I always review lockfiles and adjust them if I see deduplication potential. I think both viewpoints are perfectly valid but I would insist in this case because 15.7.2 had a critical bug fix.